contents/tutorials/customer-facing-analytics.md

```diff
@@ -0,0 +1,369 @@
+---
+title: How to set up customer-facing analytics with PostHog, Next.js, and Tremor
+date: 2023-03-25
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-9.png
+topics: ['insights']
+---
+
+If your users have their own users, sometimes your users want analytics about their users. For example, if you are a host, content platform, or some other type of B2B2C product, your users might want to know their traffic or usage metrics. Customer-facing analytics are analytics you capture and display to your users to fulfill this need.
+
+This tutorial shows you how to set up customer-facing analytics using PostHog and its API, Next.js, and Tremor (a React visualization library). You need a PostHog instance ([sign up for free](https://app.posthog.com/signup)) as well as a way to filter your analytics for an individual user, such as a user or group property (like name, domain, ID).
+
+> Thanks to [Mintlify](https://mintlify.com/) for the inspiration for this tutorial.
```

Comment:
I'm working on a Mintlify case study. I'll probably build this out and add more links when that's live.